src/tasks/publish/index.ts

```diff
@@ -168,17 +178,36 @@ async function publishTask(
     contentUri: apm.toContentUri('ipfs', contentHash)
   }
 
+  const rootAccount = await getRootAccount(bre)
+  const network = await provider.getNetwork()
+  const isTestNetwork = network.chainId > 100
+  if (!managerAddress && isTestNetwork) managerAddress = rootAccount
   const txData = await apm.publishVersion(appName, versionInfo, provider, {
     managerAddress
   })
 
-  logMain(
-    `Successfully generated TX data for publishing ${appName} version ${nextVersion}
+  if (dryRun) {
+    logMain(
+      getDryRunOutput({
+        txData,
+        appName,
+        nextVersion,
+        bump,
+        contractAddress,
+        contentHash,
+        ipfsGateway,
+        rootAccount
+      })
+    )
+  } else {
+    await bre.web3.eth.sendTransaction({
```

Comment:
Maybe we should output the main information after?

Reply:
It will output a summary for both dry runs and executions with 4ee9f0a